src/Encamina.Enmarcha.Email.Abstractions/SmtpClientOptions.cs

```diff
@@ -12,6 +13,11 @@ public class SmtpClientOptions : INameable
 {
     private string? name = null;
 
+    /// <summary>
+    /// Gets or sets a value indicating whether authentication is not required to connect to the SMTP service.
+    /// </summary>
+    public bool AuthenticationNotRequired { get; set; }
```

Comment:
Maybe, It could have a default value of false

Reply:
Actually, it is the opposite. Previous versions were always doing authentication. So, to keep previous behavior it should be `true`.

Reply:
I would add a `<remarks>` section in the comments to indicate the default value.

Reply:
I got confused with the `Authentication**Not**Required`.

Sometimes negations are hard to read 😅

If we left the name of the property as-is, then it should be as @albertodiazencamina  says, and the default value should be `false` to keep backwards compatibility. Otherwise, if we choose to change the name of the property to its positive form (`AuthenticationRequired`) then the default value should be `true`. 

Anyways, I would add the `<remarks>` section with this specification.

Reply:
Finally, I changed `AuthenticationNotRequired` to `AuthenticationRequired` to avoid confusion :) It's now `true` by default, and I've added that to the `remarks`.